<launch>
    <arg name = "node_name" default = "swarm_capture_node"/>

    <arg name = "name_space" default = "ns_000"/>
    <arg name = "uav_id" default = "000"/>
    <arg name = "num_of_uav" default = "2"/>

    <arg name = "height" default = "5"/>
    <arg name = "radius" default = "3"/>

    <arg name = "bias_x" default = "0"/>
    <arg name = "bias_y" default = "3"/>

    <node name = "$(arg node_name)" ns = "$(arg name_space)" pkg = "swarm_offb_pkg" type = "capture_pid" output = "screen">

      <param name = "uav_id" type = "string" value = "$(arg uav_id)"/>
      <param name = "num_of_uav" value = "$(arg num_of_uav)"/>
      <param name = "height" type = "double" value = "$(arg height)"/>
      <param name = "radius" type = "double" value = "$(arg radius)"/>

      <param name = "bias_x" type = "double" value = "$(arg bias_x)"/>
      <param name = "bias_y" type = "double" value = "$(arg bias_y)"/>
  </node>
</launch>